frontend/talentsearch/resources/js/components/educationExperienceForm/EducationExperienceForm.tsx

```diff
@@ -0,0 +1,198 @@
+import React from "react";
+import { useWatch } from "react-hook-form";
+import { useIntl } from "react-intl";
+import { Input } from "@common/components/form/Input";
+import { Select } from "@common/components/form/Select";
+import { Checkbox } from "@common/components/form/Checkbox";
+import { errorMessages } from "@common/messages";
+import { enumToOptions } from "@common/helpers/formUtils";
+import {
+  getEducationStatus,
+  getEducationType,
+} from "@common/constants/localizedConstants";
+import { EducationType, EducationStatus } from "../../api/generated";
+
+export const EducationExperienceForm: React.FunctionComponent = () => {
+  const intl = useIntl();
+  const isCurrent = useWatch({ name: "current-role", defaultValue: false });
+
+  // validation to test that end date is after start date, to be place into {rules} for end date input?
+  // validate: {endDateCheck: end => dateValidation(end)}
+
+  // just using this to visualize form value structure
+  // const logging = (e: any) => {
+  //   const formValue = document.getElementById("start-date");
+  //   console.log(formValue.value);
+  // };
+
+  // function to try
+  // const dateValidation = (end: string) => {
+  //   const startValue = document.getElementById("start-date");
+  //   return end > startValue.value;
+  // };
+
```

Comment:
@petertgiles 
Thinking of how to validate the end date criteria, dug around on the react-hook-form site. Got an idea for what to try 
I'm a rookie developer so please expect things to take longer than expected

Reply:
No worries @vd1992 we're all learning together. 😄 Something like this might work for you: https://github.com/GCTC-NTGC/gc-digital-talent/blob/8666dd3772cbaa1ab773b029dccea336b6039757/frontend/admin/resources/js/components/poolCandidate/CreatePoolCandidate.tsx#L442

Reply:
Thank you! I'll try it out

Reply:
Am trying out this, currently errors on load in storybook with message formValue is null

<pre>const dateValidation = () => { 
    let formValue; 
    formValue = document.getElementById("start-date") as HTMLFormElement;
    formValue = formValue.value;
    const dateChanged = Date.parse(formValue);
    return dateChanged;
  };
</pre>

<pre>
rules={
                    isCurrent
                      ? {}
                      : {
                          required: intl.formatMessage(errorMessages.required),
                          min: {
                            value: dateValidation(),
                            message: intl.formatMessage(
                              errorMessages.futureDate,
                            ),
                          },
                        }
                  }
</pre>

Reply:
Yessss, progress

Reply:
For this, react-hook-form provides a [watch feature](https://react-hook-form.com/api/useform/watch/) which is perfect for this. You can watch the value of the start date input, and then set the min to that value like [this](https://github.com/GCTC-NTGC/gc-digital-talent/blob/8fc4e3fdabbd4bd22ea8b0d69bb81167d6d7a1c3/frontend/admin/resources/js/components/classification/CreateClassification.tsx#L165-L173).

```js
const watchStartDate = watch("startDate");

...
...
...

rules={{
  required: intl.formatMessage(errorMessages.required),
  min: {
    value: watchStartDate,
    message: intl.formatMessage(errorMessages.futureDate),
  },
```

Reply:
that's way smoother than what I did, thanks